controllers/topics.controller.js

```diff
@@ -1,11 +1,11 @@
 const { readAllTopics } = require("../models/topics.model");
 
 exports.getTopics = (req, res, next) => {
-  console.log("hello from the controller");
+  //console.log("hello from the controller");
 
   readAllTopics()
     .then((topics) => {
-      console.log(topics);
+      //console.log(topics);
       res.status(200).send({ topics });
```

Comment:
console.log's/comments that are not needed should be removed before pushing your code up to github